src/App.vue

```diff
@@ -273,22 +281,25 @@ export default class App extends Mixins(
       sessionStorage.removeItem('NR_DATA')
     }
 
-    // if there is stored legal type for an IA then incorporate/register it now
+    // if there is stored legal type and request action cd, try to continue
     const legaltype = sessionStorage.getItem('LEGAL_TYPE')
-    if (legaltype && this.isAuthenticated) {
+    const requestActionCd = sessionStorage.getItem('REQUEST_ACTION_CD')
```

Comment:
This here was an old bug that I fixed on the way.

Basically, when trying to do an action without being logged in, after logging in, it wasn't working at all. The reason is previously, we're using the getters like `this.isNewBusiness` but on load, the request action cd isn't in the store so all the getters are null.

Now, I'm storing the request action cd in session storage when the user tries to do an action and setting that in the store. Now, these getters properly work.

Reply:
These fields are put in session storage because there might be a redirect between now and when the fields are needed?

Reply:
Yes, exactly. When we get back after logging, these are used to know exactly what the user was trying to do (what action and what was the legal etc..)